Documentation/ColumnsConfig/Type/Password/_Properties/_Autocomplete.rst.txt

```diff
@@ -7,9 +7,9 @@
     :Scope: Display
     :Default: false
 
-    Controls the `autocomplete` attribute of a given input field. If set to true,
-    adds attribute :php:`autocomplete="on"` to the input field allowing browser
-    auto filling the field:
+    By default, the :php:`autocomplete=new-password` attribute will be added to the
```

Comment:
The way I read this is by default, NO autocomplete is used:

`<f:form.password>` -> no autocomplete (default 'false')
`<f:form.password autocomplete="0">` -> no autocomplete (default 'false')
`<f:form.password autocomplete="off">` -> autocomplete="new-password"
`<f:form.password autocomplete="1">` -> autocomplete="new-password"
`<f:form.password autocomplete="current-password">` -> autocomplete="current-password"

(I haven't verified this though)

Reply:
We are in TCA so we have no viewhelper here. The sentence is written like this in the changelog-entry and should be true unless there is an error in the changelog

Reply:
Oohh. I'm sorry for getting totally misled here.